server/adapters/snowflake.go

```diff
@@ -328,6 +363,39 @@ func (s *Snowflake) columnDDL(name string, column Column) string {
 	return fmt.Sprintf(`%s %s`, reformatValue(name), sqlColumnTypeDDL)
 }
 
+func (s *Snowflake) ValidateWritePermission() error {
```

Comment:
After moving test eventContext creation outside storage, this func can be omitted.

Reply:
Function was removed